addons/project_account/tests/test_project_profitability.py

```diff
@@ -47,3 +47,46 @@ def test_project_profitability(self):
             },
             'The profitability data of the project should return the total amount for the revenues and costs from tha AAL of the account of the project.'
         )
+
+    def test_project_profitability_with_manufacturing_orders(self):
+        #  When the analytic account has some manufacturing orders , they should be counted only once
+        mrp_account = self.env['ir.module.module']._get('mrp_account')
+        if mrp_account.state != 'installed':
+            self.skipTest("mrp_account module is not installed")
```

Comment:
Why not creating the test inside `project_mrp` module? 🧐 
https://github.com/odoo/odoo/blob/f2422345fc0fc1e38485698a557c88c2c200f57c/addons/project_mrp/__manifest__.py#L10

Reply:
I moved the test to project_mrp , but I still need to check that `mrp_account`  and `project_account` are installed right ?